server/db.js

```diff
@@ -11,6 +11,7 @@ const pool = new Pool({
 		: { rejectUnauthorized: false },
 });
 
+
```

Comment:
Since this file wasn't really changed, you can delete this line to prevent merge conflicts.